interface/docker/postgres.conf

```diff
@@ -0,0 +1,750 @@
+# -----------------------------
```

Comment:
This looks like it came from an upstream distribution. Is it verbatim from upstream?

If not and you have changed some things, it might be nice to commit the upstream one first so your changes are in the history and an update would not be practical.